### PR TITLE
Updating html landing page copy / icon

### PR DESCRIPTION
**Pull request by Stormtv**
## High Level Overview of Change
Adds the correct PayID Logo and favicon. Reworks the link collection as the getting started link and the landing page link for the docs are now the same.

### Context of Change
- New Logo
- New Favicon
- Refactor Links

### Type of Change
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After
Screen Shot 2020-06-26 at 1 31 04 PM

## Test Plan
N/A

## Future Tasks
PayID Server Admin Portal?

### CODE REVIEW

File: src/html/index.html [hunk 1]
```diff
@@ -256,7 +180,7 @@ <h1>Welcome to your <span>PayID App</span></h1>
             );
           "
         >
-          Getting Started with PayID Guide
+          Learn more at PayID.org
```
Comment (hbergren) [inline src/html/index.html:183]: If we're switching this to the `payid.org` site, can we switch the "Develoepr Docs" link to the "Getting Started" page?

Comment (Stormtv) [reply src/html/index.html:183]: Made this change

File: src/html/index.html [hunk 2]
```diff
@@ -271,7 +195,7 @@ <h1>Welcome to your <span>PayID App</span></h1>
             );
           "
         >
-          Developer Docs
+          Getting Started with PayID Guide
```
Comment (hbergren) [inline src/html/index.html:198]: I can't comment on the actual link, but above where it says `docs.payid.org/payid-overview` it should just link to `docs.payid.org`.

Comment (Stormtv) [reply src/html/index.html:198]: wow that was the main reason I did this 🤦

**Pull request merged**